test/integration/framework/src/siftool/eth.py

```diff
@@ -242,6 +247,15 @@ def _send_raw_transaction(self, smart_contract_call_obj, from_addr, tx_opts=None
         txhash = self.w3_conn.eth.send_raw_transaction(signed_tx.rawTransaction)
         return txhash
 
+    def wait_for_all_transaction_receipts(self, tx_hashes: Sequence[HexBytes], sleep_time: int = 5,
```

Comment:
`w3_conn.eth.wait_for_transaction_receipt` returns `TxReceipt` , should we return `Sequence[TxReceipt] ` to be more precise?

also, add return type hint for `wait_for_transaction_receipt`

Reply:
Agree, fixed.